### ISSUE TITLE
RN 0.60 support

### ISSUE THREAD

**Issue by uc-asa** [kfiroo/react-native-cached-image#168]
Please add support for RN version 0.60. They deprecated  NetInfo from react-native.

It was showing an error.

**Comment by bogeresiraje** [comment]
yeah. But one an currently first install "@react-native-community/react-native-netinfo", then browse >[your-project]>node_modules>react-native-cached-image>CachedImage.js. 

1. Delete { **NetInfo** } that is destructured from **ReactNative**
2. Then import **NetInfo** from "**@react-native-community/react-native-netinfo**".

It worked for me

**Comment by ThomasFindlay** [comment]
@bogeresiraje it did work, but no one obviously should be modifying node_modules like that as everything will be lost on new install. This would require running a script after every install to replace this line.

**Comment by ElvisChiang** [comment]
@ThomasFindlay use [patch-package](https://github.com/ds300/patch-package/issues) to do the dirty thing

**Comment by changweining** [comment]
Bro，Change to another repo。
